src/amber/server/server.cr

```diff
@@ -67,9 +67,16 @@ module Amber
         exit
       end
 
-      logger.info "Server started in #{colorize(Amber.env, :yellow)}."
-      logger.info colorize("Startup Time #{Time.now - time}\n\n", :white)
-      server.listen(settings.port_reuse)
+      loop do
```

Comment:
The loop here feels odd to me since the server.listen creates something similar to listen for requests. I would suggest to just rescue the error for now  without the auto boot.

Reply:
Rescuing the error does not good what so ever if we don't restart the server. This is the same as retry in ruby.

```rb
begin
  #do things
rescue
  retry
end
```

Reply:
This is how @asterite recommended doing retry's in crystal.

Reply:
@drujensen agreed that we shouldn't let DDOS's crash the server and require manual restart. This is the obviously way to do that unless we want to force everyone to configure a shell script to do it. CTRL-C still kills the server as well as killing the PID.

Reply:
I agree about the DDOS and Retry as long as we rescue a specific error. There is no `retry` keyword in Crystal? That's sad if it doesn't